.github/workflows/docker-image.yml

```diff
@@ -26,10 +26,14 @@ jobs:
         id: ref-name
         run: echo ::set-output name=ref::${GITHUB_REF#refs/*/}
 
+      - name: downcase REPO
+        run: |
+          echo "REPO=${GITHUB_REPOSITORY,,}" >>${GITHUB_ENV}
```

Comment:
I don't really understand why this is showing up in the diff?

Reply:
That was me trying to fix the issue with CI not liking any uppercase letters in my repo name. It didn't work though, so I will revert so it disappears.

Reply:
This is being fixed in https://github.com/libp2p/rust-libp2p/pull/4421 :)

Reply:
Good to hear! I've reverted my attempt to fix it =)